objstore: Add uploaded TSDB bytes metric (#66)
thanos-io/objstore: CHANGELOG.md

- [#61](https://github.com/thanos-io/objstore/pull/61) Add OpenTelemetry TracingBucket.
    > This also changes the behaviour of `client.NewBucket`. Now it returns, uninstrumented and untraced bucket.
    You can combine `objstore.WrapWithMetrics` and `tracing/{opentelemetry,opentracing}.WrapWithTraces` to have old behavior.

> - [#66](https://github.com/thanos-io/objstore/pull/66) Add for the number of total uploaded blocks.

Reviewer: ```suggestion
- [#66](https://github.com/thanos-io/objstore/pull/66) Add `objstore_bucket_operation_written_bytes_total` that counts the number of total bytes per written to the bucket operation.
```

> - [#66](https://github.com/thanos-io/objstore/pull/66) Add `objstore_bucket_operation_written_bytes_total` that counts the number of total bytes per written to the bucket operation.

Reviewer 2: This needs an update, also should mention the sizes for Get/GetRange (missed that in my PR #69)

Author: Added a new message for your PR and mine, if you could check?

Author: Added, please review.

> - [#69](https://github.com/thanos-io/objstore/pull/69) [#66](https://github.com/thanos-io/objstore/pull/66) Add `objstore_bucket_operation_transferred_bytes` that counts the number of total bytes read from the bucket operation Get/GetRange and also counts the number of total bytes per written to the bucket operation Upload.

Reviewer 3: ```suggestion
- [#69](https://github.com/thanos-io/objstore/pull/69) [#66](https://github.com/thanos-io/objstore/pull/66) Add `objstore_bucket_operation_transferred_bytes` that counts the number of total bytes read from the bucket operation Get/GetRange and also counts the number of total bytes written to the bucket operation Upload.
```

- [#61](https://github.com/thanos-io/objstore/pull/61) Add OpenTelemetry TracingBucket.
    > This also changes the behaviour of `client.NewBucket`. Now it returns, uninstrumented and untraced bucket.
    You can combine `objstore.WrapWithMetrics` and `tracing/{opentelemetry,opentracing}.WrapWithTraces` to have old behavior.

> - [#69](https://github.com/thanos-io/objstore/pull/69) Add `objstore_bucket_operation_transferred_bytes` that counts the number of total bytes read from the bucket operation Get/GetRange.

Reviewer 2: Let's just keep the one line

```suggestion
```

Author: Done.
